store/localcachelayer/channel_layer.go

```diff
@@ -132,3 +149,25 @@ func (s LocalCacheChannelStore) GetPinnedPostCount(channelId string, allowFromCa
 
 	return count, nil
 }
+
+func (s LocalCacheChannelStore) Get(id string, allowFromCache bool) (*model.Channel, *model.AppError) {
+	return s.get(id, false, allowFromCache)
```

Comment:
why not adding `get` code directly here, i didn't see any other use, so this is adding layers without any benefit